Project-II-Calculator/calculator/src/components/ButtonComponents/Button.css

```diff
@@ -0,0 +1,43 @@
+.actionButton {
+  box-sizing: border-box;
```

Comment:
`box-sizing: border-box` you would only typically find on the body tag or as a wildcard (*) selector: 

```
* {
       box-sizing: border-box;
   }
```

You want it to pertain to the whole file or app if you are going to use it so your CSS doesn't get all wonky.